content/docs/alerting/clients.md

```diff
@@ -39,10 +39,10 @@ The labels are used to identify identical instances of an alert and to perform
 deduplication. The annotations are always set to those received most recently
 and are not identifying an alert.
 
-Both timestamps are optional. If `startsAt` is omitted, the current time
-is assigned by the Alertmanager. `endsAt` is only set if the end time of an
-alert is known. Otherwise it will be set to a configurable timeout period from
-the time since the alert was last received.
+Both timestamps are optional. If `startsAt` is omitted, AlertManager will set
+it to `endsAt` or the current time if `endsAt` is missing. `endsAt` is only
```

Comment:
> `endsAt` is only set if the end time of an alert is known.

This doesn't really say anything, known by whom?

It should be clear that it's okay to set it on alerts that are currently firing.